frontend/src/common.js

```diff
@@ -0,0 +1,3 @@
+import './helpers-copy-paste';
+import './helpers-feedbacks';
+import './load-bar';
```

Comment:
Great. 

I believe we also need front state machine and inpi pdf download

Can you add a detailled comment to explain the purpose of this file and distinguish it from `/index` ?

Reply:
https://github.com/etalab/annuaire-entreprises-site/pull/726/commits/f6c8ff2c5057f6f16de9d16004aebd5913743559#diff-ff1d61de8655c32a447df582516565f722b8f17d0054fade55829de6800d9a2fR1-R10